HunterPie.UI/Architecture/Converters/QuestLifeToVisibilityConverter.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Globalization;
+using System.Windows;
+using System.Windows.Data;
+using Converter = System.Convert;
+
+namespace HunterPie.UI.Architecture.Converters;
+
+public class QuestLifeToVisibilityConverter : IValueConverter
+{
+    public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        int questlife = Converter.ToInt16(value);

```

Comment:
Not sure why you're converting it to a 16 bits integer here, you can just hard cast it.

```suggestion
        int questlife = (int)value;

```